.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "jest/globals": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+        {
+            "files": ["/*.cy.js", "/*.test.ts"],
```

Comment:
It looks like you have merged config for cypress and jest instead of giving these their own block.